### DISCUSSION TITLE
Setting IPv6 Address

### DISCUSSION THREAD

**Discussion by hupiao** [Wifx/gonetworkmanager#42] [Q&A]
I want to know how to configure IPv6 addresses, but the example provided only has methods for configuring IPv4 addresses. Could you please show how to configure IPv6 addresses?

**Comment by mullerch** [comment]
I don't have a full example. However, you can start from the `static_connect` example. The IPv6 properties are detailed in the NetworkManager's documentation at https://networkmanager.dev/docs/api/latest/nm-settings-dbus.html#id-1.2.9.4.19.
